plugins/container_logs.yaml

```
@@ -0,0 +1,116 @@
+version: 0.0.1
+title: Kubernetes Container Logs
+description: Log parser for Kubernetes Container logs
+parameters:
+  - name: log_paths
+    type: "[]string"
+    description: A list of file glob patterns that match the file paths to be read
+    default:
+      - "/var/log/containers/*.log"
+  - name: exclude_file_log_path
+    type: "[]string"
+    description: A list of file glob patterns to exclude from reading
+    default:
+      - "/var/log/containers/observiq-*-collector-*"
+  - name: start_at
+    type: string
+    description: At startup, where to start reading logs from the file.
+    supported:
+      - beginning
+      - end
+    default: end
+
+template: |
+  receivers:
+    filelog:
+        include:
+          {{ range $fp := .log_paths }}
+            - '{{ $fp }}'
+          {{end}}
+        start_at: {{ .start_at }}
+        log_type: "k8s.container_logs"
```

I think this needs to be nested under attributes (https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver)
```suggestion
        attributes:
            log_type: "k8s.container_logs"
```
Something we need to figure out with our plugin receiver is how to make it more robust against these types of configs; It just drops fields if they aren't valid options instead of giving an error.

---

Yeah you're right. went too fast.